### ISSUE TITLE
Summary - arrow visibility 

### ISSUE THREAD

**Issue by m0jimo** [quasarframework/quasar-ui-qmarkdown#208]
There is some problem with viewing an arrow for expanding summary details content.

**Describe the bug**
Arrow is not visible in some cases (screenshot from [interactive demo](https://quasarframework.github.io/quasar-ui-qmarkdown/demo)).
![image](https://user-images.githubusercontent.com/37825539/119325218-f8be7800-bc80-11eb-83a5-e755fa17df4d.png)
vs 
![image](https://user-images.githubusercontent.com/37825539/119325389-1c81be00-bc81-11eb-8a83-ca11341609ad.png) 
I have two applications - one in Electron which shows the arrow the other application in SPA which doesn't - I can't spot the differences in the hierarchy of components - but doesn't mean it is influenced by the parent. 

**To Reproduce**
1. Go to [interactive demo](https://quasarframework.github.io/quasar-ui-qmarkdown/demo)
2. Insert
```md
<details>
<summary>Click to reveal content</summary>

This is the hidden content
::: tip
Use any other content inside.
:::

</details>
```
3. Can you see an arrow?
4. To show arrow disable `display: block` from the element (inherited from Quasar styles)
![image](https://user-images.githubusercontent.com/37825539/119325718-72eefc80-bc81-11eb-801d-c014ce60a3d1.png)
and the arrow appears
![image](https://user-images.githubusercontent.com/37825539/119325739-797d7400-bc81-11eb-9b08-67e19bbc0d22.png)

**Expected behavior**
Show arrow to be clear that the content can be revealed.

**Screenshots**
see above.

**Desktop (please complete the following information):**
Win10
Visible inside Electron
Not visible in Chrome
but in both cases, the `summary` tag has `display: block`.

**Additional context**
I tried to change the `display: xxx` to rewrite the original CSS but no success.
```
<style>
.q-markdown summary {
  display: inline!important; /* unset doesn't work either */
}
</style>
```

**Comment by hawkeye64** [comment]
Actually, the sass would look like this:

```css
.q-markdown
  details > summary
    display: list-item
```

I'll have a fix out shortly.

**Comment by hawkeye64** [comment]
Btw, thanks for finding this! :heart:
Fix will be in v1.4.3
![image](https://user-images.githubusercontent.com/10262924/119408242-a2623100-bca2-11eb-8f39-b0db5ab9b78f.png)
